src/atoms/Link/__tests__/Link.web.test.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react';
+import Link from '../index';
+import { renderWithTheme } from '../../../_helpers/testing';
+
+describe('<Link />', () => {
+  it('renders a link with default props', () => {
+    const displayText = 'Displaying some link';
+    const { container } = renderWithTheme(<Link>{displayText}</Link>);
+    expect(container).toMatchSnapshot();
+  });
+  it('renders a link with disabled prop', () => {
+    const displayText = 'Displaying some link';
+    const { container } = renderWithTheme(<Link disabled>{displayText}</Link>);
+    expect(container).toMatchSnapshot();
```

Comment:
Both of these tests should go under [this](https://github.com/razorpay/blade/pull/92/files#diff-c819286a0f7ab952e5b9186bb2fdca82R62) test suite?